packages/design-system/src/components/Icons/MenuIcon.tsx

```diff
@@ -1,24 +1,22 @@
 import React from 'react';
+import SvgIcon, { IconCommonProps } from './SvgIcon';
 
-/**
- * MenuIcon component which returns the menu SVG as a React component
- */
+const defaultProps = {
+  className: 'svg-inline--fa fa-bars fa-w-14',
+  title: 'Menu Icon',
+  viewBox: '0 0 448 512',
+};
+
+function MenuIcon(props: IconCommonProps): React.ReactElement {
+  const iconCssClasses = `ds-c-icon--menu ${props.className || ''}`;
 
-export function MenuIcon({ className }) {
   return (
-    <svg
-      aria-hidden="true"
-      data-prefix="fas"
-      data-icon="bars"
-      className={`svg-inline--fa fa-bars fa-w-14 ${className}`}
-      xmlns="http://www.w3.org/2000/svg"
-      viewBox="0 0 448 512"
-    >
+    <SvgIcon {...defaultProps} {...props} className={iconCssClasses}>
```

Comment:
[fix] Because the `className={iconCssClasses}` attribute is after the `defaultProps` are spread, the default value for `className` will never be used. <SvgIcon> will always use the value of `iconCssClasses`.

Consider either:
1. Updating `iconCssClasses` to include `svg-inline--fa fa-bars fa-w-14`, if those classes are needed
2. Making the default value of `className: ''` if they are not

Reply:
Oh shoot I forgot to update this file thats a copy-paste-forgot to finish thing, will fix!

Reply:
This is a place where the `classNames` function can be used to avoid messy or complicated interpolated strings.